src/EditorFeatures/Test/Diagnostics/DiagnosticAnalyzerServiceTests.cs

```diff
@@ -96,6 +93,81 @@ public async Task TestHasSuccessfullyLoadedBeingFalseWithCompilerAnalyzerFSAOn()
             await TestAnalyzerAsync(workspace, document, new CSharpCompilerDiagnosticAnalyzer(), CompilerAnalyzerResultSetter, expectedSyntax: true, expectedSemantic: false);
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
```

Comment:
💡 You can replace these with `[CombinatorialData]` which will automatically provide `true` and `false`.

Reply:
It would be nice to have an analyzer/fix for this :-). I'll retain this as is because the combinatorial data here is just 2 values.